grails-app/utils/org/pih/warehouse/PaginatedList.groovy

```diff
@@ -0,0 +1,17 @@
+package org.pih.warehouse
+
+class PaginatedList {
```

Comment:
overall this looks good to me, however I'm wondering if we can make this "type safe", so to do it like:
```groovy
class PaginatedList<T> {
   @Delegate
   List<T> list;
   
   int totalCount;
}
```
and use it like:
```groovy
return new PaginatedList<StockMovement>(stockMovements, requisitions.totalCount)
```

if it's possible, then I think it'd be a nice addition, if not, then LGTM

Reply:
sure, sounds good